force-app/main/default/lwc/rd2EntryFormScheduleSection/rd2EntryFormScheduleSection.js

```diff
@@ -557,6 +559,10 @@ export default class rd2EntryFormScheduleSection extends LightningElement {
                 }
             });
 
+        if(data[this.fields.recurringType.apiName] === RECURRING_TYPE_OPEN) {
+            data[this.fields.plannedInstallments.apiName] = null;
```

Comment:
This works great, allows the form to keep the Installment Number value if customers switch from Fixed to Open, and then decide to switch back.